mypyc/genops.py

```diff
@@ -1050,7 +1050,9 @@ def __init__(self,
 
         self.errors = errors
         self.mapper = mapper
-        self.imports = []  # type: List[str]
+        # We use an OrderedDict for this so we can maintain insertion order
+        # and do quick lookups
+        self.imports = OrderedDict()  # type: OrderedDict[str, None]
```

Comment:
Document this attribute. Maybe this would benefit from renaming -- it took me a while to figure out what it's used for.